store/migrate_test.go

```diff
@@ -312,7 +383,7 @@ type RemoteV2_3 struct {
 
 func getAllRemoteV2_3(tx *sql.Tx) ([]*RemoteV2_3, error) {
```

Comment:
Since remote haven't change until V6 this and the related struct should be changed to RemoteV2_6